tests/server_test_cases/person_note_tests.py

```diff
@@ -2630,112 +2631,166 @@ def test_search_api(self):
             config.set_for_repo('haiti', search_auth_key_required=False)
 
 
-    def test_sms_api(self):
-        """Tests the behavior of SMS API."""
+    def verify_sms_responses(self, test_data):
+        for test_input, exp_output in test_data.iteritems():
+          request_data = (
+              '<?xml version="1.0" encoding="utf-8"?>'
+              '<request>'
+              '    <message_text>%s</message_text>'
+              '    <receiver_phone_number>%s</receiver_phone_number>'
+              '</request>') % (test_input[1], test_input[2])
+          doc = self.go(test_input[3], data=request_data,
+                        type='application/xml')
+          assert exp_output[0] == self.s.status, (
+              'Failure on status for %s, %d != %d' % (
```

Comment:
I did it that way so that the "label" for the individual test would be printed (so that you could tell which test failed), but now that I've switched to having one call to verify_sms_response for each test, that's not really an issue since you can just look at the line number.